_layouts/home.html

```diff
@@ -43,6 +43,5 @@ <h1 class="post-title-main">{{ page.title }}</h1>
 
 </div>
 
-{{site.data.alerts.hr_shaded}}
+<!-- {{site.data.alerts.hr_shaded}} -->
```

Comment:
I'm not familiar with what this line would do. But if we don't need it, perhaps we should remove it.

Reply:
Right now i don't remember exactly. Because i edited it 2 month ago. I believe it was in the wrong position, or in this case, in the wrong "div". I moved it.

Reply:
ok. if you don't think we need it anymore, we can remove it instead of commenting out. Thanks.